content/ConnectingToDatabase/ConnectingToDatabase.facts.md

```diff
@@ -0,0 +1,60 @@
+### Facts for Connecting to Database lesson:
+
+`PHP Data Objects (PDO)` defines an interface in PHP to access certain databases
+
+`PDO` class represents a connection between a database server and PHP.
+
+`PDO_PGSQL` is a driver of `PHP Data Objects (PDO) Interface` that allows access between PHP and PostgreSQL database.
+
+`PDO_PGSQL DSN` is the PostgreSQL data source name that carries the database parameters to establish a connection. It is composed of the following DSN parameters:
+
+- `pgsql:` is the DSN prefix.
+    
+- `host` refers to the server’s hostname of the PostgreSQL database.
+
+- `port` refers to the port number of the PostgreSQL database server.
+    
+- `dbname` is the database name.
+
+- `user` refers to the username of the user.
+    
+- `password` refers to the password that is set by the user.
+
+The example code below shows how PHP uses the `PDO_PGSQL` driver, `PDO_PGSQL DSN`, and `PDO` class to connect to the PostgreSQL database.
```

Comment:
I didn't see any usage of the `PDO_PGSQL` below. Learners might get confuse because no where on the code below did we refer to `PDO_PGSQL`